frappe/handler.py

```diff
@@ -66,7 +64,17 @@ def execute_cmd(cmd, from_async=False):
 
 	return frappe.call(method, **frappe.form_dict)
 
+def get_overridden_method(cmd):
+	for hook in frappe.get_hooks("override_whitelisted_methods", {}).get(cmd, []):
+		# override using the first hook
+		return hook
+
+	return cmd
+
 def is_valid_http_method(method):
+	if frappe.flags.in_safe_exec:
+		return
```

Comment:
This has been probably fixed in https://github.com/frappe/frappe/pull/14835 as well, but waiting for some comments there.

PS: this PR will clash with that one 😅